e2e/bug10483/src/bug10483.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+function doFoo(mixed $filter): void {
+    \PHPStan\Testing\assertType('non-falsy-string|false', filter_var("no", FILTER_VALIDATE_REGEXP));
```

Comment:
Alright so I found out that the usual result on PHP versions where this constant exists is the same as here. Which makes me a bit suspicious. Can you explain that? We surely must be losing information when we return the unknown constant somewhere, right?